src/constants.ts

```diff
@@ -2,8 +2,10 @@ import config from "../config.json";
 
 export const PLUGIN_NAME = "Starknet";
 export const ABI_SUFFIX = "_abi.json";
-export const CAIRO_ASSEMBLY_SUFFIX = ".casm";
-export const CARGO_FILE = "Cargo.toml";
+export const CAIRO1_ASSEMBLY_SUFFIX = ".casm.json";
```

Comment:
Does starknetjs use .casm extension? Or you meant Backwards compatibility with previous plugin version?
I changed it because output is in json format. I remember i wanted to view file in vscode and I just got .txt view of a file with bunch of highlighted errors so i decided to change an extension so it doesn't complain. Or maybe I'm missing something? Maybe there's indeed such thing as .casm format and vscode just doesn't support it yet

Reply:
Yes, starknetjs uses .casm extension. In general, it's more reasonable to keep extensions similar to keep consistency and avoid issues when working on/with different tools.

Reply:
Can we change it in starknetjs instead? :) It makes more sense for me to add .json since it is a json file

Reply:
We could do that, but from what I can see, OZ's Nile compiler will also use the .casm extension and perhaps other tools. Only concerned in case the mismatch leads to problems.